debugger/ptraceLib/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-find_library(DL_LIB NAMES dl)
-add_library(dl SHARED IMPORTED)
-set_target_properties(dl PROPERTIES IMPORTED_LOCATION ${DL_LIB})
+find_library(DL_LIB NAMES dlib)
```

Comment:
Since the `dl` library should probably always exist (can we verify this?) we can probably just check if this results in Not Found and error out, and make sure we do the same in our CMake package, we should definitely not be swapping to another implementation.